pkg/netid/netid.go

```diff
@@ -0,0 +1,94 @@
+package netid
+
+/*
+Accessor methods to annotate namespace for multitenant support
+*/
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+const (
+	// Maximum VXLAN Virtual Network Identifier(VNID) as per RFC#7348
+	MaxVNID = uint((1 << 24) - 1)
+	// VNID: 1 to 9 are internally reserved for any special cases in the future
+	MinVNID = 10
```

Comment:
No, this is not a new restriction. We have been using 10 as min vnid from the beginning. We don't expect namespace to have vnid between 1 to 9 but if that happens for some reason, vnid repair controller will detect this issue and won't allow starting master unless admin fixes this issue.